sys/net/gnrc/network_layer/icmpv6/echo/gnrc_icmpv6_echo.c

```diff
@@ -150,6 +150,12 @@ int gnrc_icmpv6_echo_send(const gnrc_netif_t *netif, const ipv6_addr_t *addr,
     ipv6_hdr_t *ipv6;
     uint8_t *databuf;
 
+    /* max IPv6 payload 65535 minus 8 bytes of icmp header = 65527 */
+    if (len > 65527) {
```

Comment:
Ah and can you please use e.g. `UINT16_MAX - sizeof(icmpv6_hdr_t)` instead of the magic number, then you don't need the comment ;-).

Reply:
Ok. Fixed. If this is not a problem I would like to leave this comment.